CHANGES.md

```diff
@@ -1,5 +1,7 @@
 ## Changes in 0.6.2 (in development)
 
+* Finished implementation of s3 data store
```

Comment:
Please write change logs by using the perspective of an interested user. So what is the reason and benefit of the change?